src/main/java/picard/util/IntervalListTools.java

```diff
@@ -253,38 +253,68 @@
             IntervalList act(final List<IntervalList> list, final List<IntervalList> __) {
                 return IntervalList.concatenate(list);
             }
+
+            @Override
+            public boolean takesSecondInput() {
+                return false;
+            }
         },
         UNION("Like CONCATENATE but with UNIQUE and SORT implied, the result being the set-wise union of all INPUTS, " +
                 "with overlapping and abutting intervals merged into one.") {
             @Override
             IntervalList act(final List<IntervalList> list, final List<IntervalList> __) {
                 return IntervalList.union(list);
             }
+
+            @Override
+            public boolean takesSecondInput() {
+                return false;
+            }
         },
         INTERSECT("The sorted and merged set of all loci that are contained in all of the INPUTs.") {
-            @Override
-            IntervalList act(final List<IntervalList> list, final List<IntervalList> __) {
+            @Override IntervalList act(final List<IntervalList> list, final List<IntervalList> __) {
```

Comment:
`@Override` should be on its own line, e.g. the same as the rest of this file is formatted.